util-app/src/main/scala/com/twitter/app/App.scala

```diff
@@ -313,7 +313,7 @@ trait App extends Closable with CloseAwaitably {
     deadline: Time
   ): Future[Unit] = {
     Future
-      .collectToTry(lastExits.asScala.toSeq.map(_.close(deadline)))
+      .collectToTry(lastExits.asScala.map(_.close(deadline)).toSeq)
```

Comment:
why does it matter where the toSeq is?

Reply:
ping

Reply:
I think it was part of a refactor that turned out wasn't needed, and this is a vestigial remainder of that refactor.

In other words, I don't think it matters, and pushed a commit to undo it

Reply:
thanks for clarifying.